src/objects.c

```diff
@@ -60,8 +60,8 @@ s8 D_800DC73C = 0;
 s8 D_800DC740 = 0;
 s8 gSwapLeadPlayer = FALSE;
 s8 D_800DC748 = FALSE;
-s32 D_800DC74C[2] = { 0, 0 }; // Have a feeling these are both the same array.
-s32 D_800DC754[2] = { 0, 0 };
+Vertex *D_800DC74C[2] = { 0, 0 }; // Have a feeling these are both the same array.
```

Comment:
```suggestion
Vertex *D_800DC74C[2] = { 0, 0 };
```

I presume these are infact not the same array anymore